### ISSUE TITLE
Linux: "No bottle available" for gdbm

### ISSUE THREAD

**Issue by peazncheez** [Homebrew/homebrew-core#85816]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

```shell
`brew config`
HOMEBREW_VERSION: 3.2.13
ORIGIN: https://github.com/Homebrew/brew
HEAD: 109f24fd601177dea4a754b43efe684903db2a83
Last commit: 4 days ago
Core tap ORIGIN: https://github.com/Homebrew/linuxbrew-core
Core tap HEAD: 71fef975b32cd53af3256d0e8cc4fe489f3c6d80
Core tap last commit: 2 hours ago
Core tap branch: master
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 64
Homebrew Ruby: 2.6.3 => /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.3_2/bin/ruby
CPU: 64-core 64-bit skylake
Clang: N/A
Git: 2.25.1 => /bin/git
Curl: 7.68.0 => /usr/bin/curl
Kernel: Linux 5.11.0-1017-aws x86_64 GNU/Linux
OS: Unknown
Host glibc: 2.31
/usr/bin/gcc: 9.3.0
/usr/bin/ruby: N/A
glibc: N/A
gcc@5: N/A
xorg: N/A

`brew doctor`
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: "config" scripts exist outside your system or Homebrew directories.
`./configure` scripts often look for *-config scripts to determine if
software packages are installed, and which additional flags to use when
compiling and linking.

Having additional scripts in your path can confuse software installed via
Homebrew if the config script overrides a system or Homebrew-provided
script of the same name. We found the following "config" scripts:
  /opt/pyenv/shims/python3.7-config
  /opt/pyenv/shims/python3.7m-config
  /opt/pyenv/shims/python-config
  /opt/pyenv/shims/python3-config
```

### 

- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.

### What were you trying to do (and why)?

Install `exiftool` via linuxbrew, and `gdbm` is one of the dependencies of `exiftool`. `exiftool` has been installable via linuxbrew for over a year with no issues.

### What happened (include all command output)?

```
==> Downloading https://ghcr.io/v2/linuxbrew/core/berkeley-db/manifests/18.1.40
Already downloaded: /home/circleci/.cache/Homebrew/downloads/ae724b4d70a1879f33c74bab6dd7e477c347965e04701c4766aec0548f09609f--berkeley-db-18.1.40.bottle_manifest.json
==> Downloading https://ghcr.io/v2/linuxbrew/core/berkeley-db/blobs/sha256:9935bb07d8c09bc317aa61776b12b0277b136d1d5dfb65f
Already downloaded: /home/circleci/.cache/Homebrew/downloads/527d0b9034d43711418e0ccf37eddb71b4df07e4b16bcd38935835e6b9f67b03--berkeley-db--18.1.40.x86_64_linux.bottle.tar.gz
Error: gdbm: no bottle available!
You can try to install from source with:
  brew install --build-from-source gdbm
Please note building from source is unsupported. You will encounter build
failures with some formulae. If you experience any issues please create pull
requests instead of asking for help on Homebrew's GitHub, Twitter or any other
official channels.
```

### What did you expect to happen?

`exiftool` would be successfully installed.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
`brew install exiftool`
or more directly
`brew install gdbm`
```

**Comment by carlocab** [comment]
@Homebrew/linux

**Comment by peazncheez** [comment]
Looks like the linuxbrew build for `gdbm` started failing earlier today - https://github.com/Homebrew/linuxbrew-core/actions/runs/1267433694

Some recent relevant changes - https://github.com/Homebrew/linuxbrew-core/pull/24458, https://github.com/Homebrew/linuxbrew-core/commit/7efa44a0d1da60e0fff00ae39760081893964131

**Comment by iMichka** [comment]
Fixed by 41b84d7aa59a6ce1867907937eb566700b85d638
